qa/suites/upgrade-clients/client-upgrade-octopus-pacific/octopus-client-x/rbd/3-workload/rbd_notification_tests.yaml

```diff
@@ -8,25 +8,23 @@ tasks:
       env:
         RBD_FEATURES: "61"
   - workunit:
-      #The line below to change to 'pacific'
-      branch: master
+      branch: pacific
```

Comment:
@ideepika I was expecting this PR would just set DISABLE_UPDATE_FEATURES variable to skip the test. But I see more changes like this one. Could you explain why they are here?

Reply:
removed 2nd commit, in favor of: https://github.com/ceph/ceph/pull/41513#issuecomment-849705557